src/TimeManager.jl

```diff
@@ -63,4 +72,69 @@ Returns `true` if the current date is equal to or exceeds the saved first of the
 """
 trigger_callback(cs::CoupledSimulation, ::Monthly) = cs.dates.date[1] >= cs.dates.date1[1] ? true : false
 
+
+abstract type CouplerCallback end
+
+"""
+    do_nothing(::Interfacer.CoupledSimulation, _)
+
+This is a helper callback function that does nothing.
+"""
+do_nothing(::Interfacer.CoupledSimulation, _) = nothing
+
+"""
+    HourlyCallback{FT}
+
+This is a callback type that triggers at intervals of 1h or multiple hours.
+"""
+@kwdef struct HourlyCallback{FT} <: CouplerCallback
+    dt::FT = FT(1) # hours
```

Comment:
We should use `DocStringExtensions` and add descriptions so the API docs automatically pick up property information.